cocos/particle/models/particle-batch-model.ts

```diff
@@ -170,14 +170,20 @@ export default class ParticleBatchModel extends scene.Model {
         ));
         const vBuffer: ArrayBuffer = new ArrayBuffer(this._vertAttribSize * this._capacity * this._vertCount);
         if (this._mesh && this._capacity > 0) {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-enum-comparison
             let vOffset = (this._vertAttrs![this._vertAttrs!.findIndex((val): boolean => val.name === AttributeName.ATTR_TEX_COORD)] as any).offset;
```

Comment:
please define val.name as `AttributeName` instead of `string`

don't disable eslint rule, try to fix it

Reply:
val.name is from gfx